2017/presentations/with_python_security/text.md

```diff
@@ -0,0 +1,519 @@
+# with python: security() - Piotr Dyba
+
+
+#### Disclaimers:
+The views and opinions expressed in this article focus on the developer's perspective an are solely those of the author.
+I have received no payments for mentioning any of the open-source or commercial products mentioned in this article.
+
+
+## What is cybersecurity and why it's so important nowadays?
+
+Cybersecurity encompasses means and protocols to defend resources and devices (computer systems, IoT devices, smartphones) from disruption, damage or theft.
+It is not only about firewalls and antivirus software, but rather it starts with the hardware,
+including policies for handling computers and servers at a company,
+access rights to server rooms and even which devices are allowed to be brought into the company building. For instance, it is common to ban external pen-drives.
+To be able to assume that our hardware is safe, we need to harden our software on both workstations and servers and impose
+proper policies for the users, data handling, access.
+
+[Data leaks in time](http://www.informationisbeautiful.net/visualizations/worlds-biggest-data-breaches-hacks/)
+
+Looking at the number and severity of leaks over time, it is easy to observe a huge increase in their occurences over the years and severity rising from 1 000 to above 1 000 000 000 user's data.
```

Comment:
W drugiej części zdania "severity" dałbym przed occurences, bo teraz to brzmi, że to severity urosło od 1K do 1KK.